platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/annotations/Hole.java

```diff
@@ -12,7 +12,7 @@
 /**
  * Encapsulates a {@link List} of {@link LatLng} points defining a hole
  */
-public class Hole extends ArrayList<LatLng> implements Parcelable {
+class Hole extends ArrayList<LatLng> implements Parcelable {
```

Comment:
since we are removing Hole.java from public API, is there a benefit of keeping this class around?